staging/src/k8s.io/cloud-provider/OWNERS

```diff
@@ -0,0 +1,48 @@
+approvers:
+- mikedanese
+- dims
+- wlan0
+- andrewsykim
+- cheftako
+reviewers:
+- thockin
```

Comment:
Might be worth trimming this list for faster reviews, but :man_shrugging:

Reply:
Good point.